docs/topics/secured-mission-common-interaction.adoc

```diff
@@ -45,3 +115,20 @@ NOTE: This is the master realm public key from the Secured booster's {RHSSO} ser
 
 . Paste the `token` output from the client output in the _Encoded_ box.
 . You will see `Signature Verified` on the debugger page.
+
+== Using the Web Interface
+The secured endpoint also contains a web interface to interact with it. Access it using the xref:secured-endpoint[endpoint address].
+
+When using the web interface, you will go through the following steps to see how security is enforced, how authentication happens, and how the token is requested and attached to an authenticated invocation.
+
+. Open the web interface and click on _Invoke_. This sends an unauthenticated request to the Greeting service. 
+. The services responds with an `HTTP 401 Unauthenticated` status code.
+. Click _Login_ to authenticate yourself against {RHSSO}. This redirects you to the SSO server and requests your credentials. 
+. Log in as xref:sso-users[the _Alice_ user].
+. Once authenticated you are redirected to the web interface, which includes a bearer token shown in the command line output at the bottom of the page.
+. Click _Invoke_ again to access the Greeting service.
+. Confirm there is no exception and JSON response payload displayed. This means the service accepted your bearer token and you are authorized access to the Greeting service.
+. Logout.
+. Click on _Invoke_. Confirm this sends an unauthenticated request to the Greeting service. 
+. Click _Login_ and log in as xref:sso-users[the _admin_ user].
+. Click on _Invoke_. Confirm this sends an authenticated request to the Greeting service, but you receive an error relating to the user being unauthorized.
```

Comment:
Should probably mention, similarly to how we mention `HTTP 401` above, that this is `HTTP 403 Forbidden`.